WTWSMS/decisions/minor_decisions.txt

```diff
@@ -4,17 +4,17 @@ decisions = {
 		only_playable = yes
 		potential = {
 			is_playable = yes
-			OR = {
-				is_feudal = yes
-				is_tribal = yes
-				is_nomadic = yes
-			}
+
 			prisoner = no
-			OR = {
-				religion_group = christian
-				religion_group = zoroastrian_group
-				religion_group = jewish_group
+
+			NOT = { 
+				religion_group = indian_group # They have their own Duwali Feasts
+				religion = zun_pagan # They have their own Sun Feasts
+				religion = zun_pagan_reformed # They have their own Sun Feasts
```

Comment:
And all the other cultures who have 60 different equivalents? This change doesn't make much sense in that regard, because the existant restrictions already did this, just that they covered everyone properly.